biz.aQute.bnd.gradle/src/aQute/bnd/gradle/BundleTaskConvention.groovy

```diff
@@ -28,9 +28,7 @@ import static aQute.bnd.gradle.BndUtils.jarLibraryElements
 import static aQute.bnd.gradle.BndUtils.logReport
 import static aQute.bnd.gradle.BndUtils.unwrap
 
-import java.util.Properties
```

Comment:
The Properties type is used in the `buildBundles` method, so this line should not be removed.

Reply:
Err, it can only be because IntelliJ flagged it as "unused" - for some reason known only to IntelliJ. I'll add it back...

Reply:
Ah, the explanation! [Groovy imports these packages by default](https://groovy-lang.org/structure.html):
```
import java.lang.*
import java.util.*
import java.io.*
import java.net.*
```